Packs/VersaDirector/Integrations/VersaDirector/VersaDirector.py

```diff
@@ -1694,9 +1694,11 @@ def create_client_header(
         if username and password:
             credentials = f"{username}:{password}"
             auth_header = f"Basic {b64_encode(credentials)}"
-            return (username, password), {"Authorization": auth_header}
+            return (username, password), {"Authorization": auth_header, 'Accept': 'application/json',
+                                          'Content-Type': 'application/json'}
```

Comment:
Are we sure all responses coming from the API would be in a JSON format? If I understand correctly it would all be affected.

Reply:
Yes they are.